src/Client.ts

```diff
@@ -244,7 +246,8 @@ export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
  * Client class initiates connection to the XMTP network.
  * Should be created with `await Client.create(options)`
  */
-export default class Client {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export default class Client<T = any> {
```

Comment:
This would be better, but also much more disruptive, if I made it `<T = unknown>`